packages/twenty-chrome-extension/src/db/types/company.ts

```diff
@@ -0,0 +1,13 @@
+export type Company = {
```

Comment:
You probably should get the types from the codegen, best practise would be to do like what's on the frontend with a `pick`